baselines/common/vec_env/vec_env.py

```diff
@@ -145,7 +145,8 @@ class VecEnvWrapper(VecEnv):
 
     def __init__(self, venv, observation_space=None, action_space=None):
         self.venv = venv
-        super().__init__(num_envs=venv.num_envs,
+        VecEnv.__init__(self,
```

Comment:
these two are equivalent in python3, are they not?

Reply:
Yep. Changing it back.